www/src/public/js/controllers/builder/main.js

```diff
@@ -1196,6 +1339,25 @@
 
                 return;
             }
+
+            //if the new item is not a runecharm, set that charm slot to default AAAAA
+            if($scope.selectedList.items[$scope.currentItemIndex].id === -5 && angular.copy(item).id != -5) {
```

Comment:
I don't think you need the angular.copy here, but test it to be sure. It's an expensive call, so it would be great if you don't need it.

Reply:
removed.